pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "arcaflow-plugin-smallfile"
+version = "0.1.0"
+description = "arcaflow plugin for smallfile"
+authors = ["arcalot"]
+license = "Apache-2.0+GPL-2.0-only"
```

Comment:
Same question about GPL 2 here as in Dockerfile

Reply:
Changed to Apache-2.0